scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -135,15 +149,28 @@ impl PrecomputedQueries {
         timestamp: i64,
         column_name: &str,
     ) -> anyhow::Result<()> {
-        run_statement(
+        let overwrite_query = match self.overwrite_queries.get_mut(column_name) {
+            Some(query) => query,
+            None => self
+                .overwrite_queries
+                .entry(column_name.to_string())
+                .or_insert(
```

Comment:
This can be changed to just `.insert(column_name.to_string(), ...)`.

Reply:
Disregard my comment, I realized that you need to obtain a reference to the element... `entry().or_insert()` sounds OK.